packages/preview/diatypst/0.2.0/template/main.typ

```diff
@@ -0,0 +1,27 @@
+#import "@preview/diatypst:0.2.0": *
+
+#show: slides.with(
+  title: "Diatypst", // Required
+  subtitle: "easy slides in typst",
+  date: "01.07.2024",
+  authors: ("Author Name"),
+
+  // Optional Styling
+  ratio: 16/9,
+  layout: "medium",
+  title-color: green.darken(60%),
+  footer: true,
+  counter: true,
```

Comment:
Can you rename the `counter` argument here too please?

Reply:
ups forgot that. should be fixed now